codec/data/assurances_extrinsic.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "anchor": "0x0cffbf67aae50aeed3c6f8f0d9bf7d854ffd87cef8358cbbaa587a9e3bd1a776",
+        "bitfield": "0x01",
```

Comment:
If we'd like to be more explicit about the actual value of the bitsequence `bitfield` (and not the octet representation, I'd change this JSON-testvector to show the following value. The size of the fixed length bitsequence is set to 2 (GP_constant_C). The suggested change below avoids any confusion about its length. That said, this is not really necessary, since the byte representation is also correct.
```
"bitfield": [
    True,
    False
],
```